cmake/modules/Hexagon.cmake

```diff
@@ -178,6 +178,15 @@ if(BUILD_FOR_HEXAGON)
     "${TVMRT_SOURCE_DIR}/hexagon/ops/*.cc"
   )
 
+  include_directories(
+    "${TVMRT_SOURCE_DIR}/hexagon/ops"
+  )
+
+  set_source_files_properties(
+    "${TVMRT_SOURCE_DIR}/hexagon/ops/conv2d_quant_hvx.cc"
+    PROPERTIES COMPILE_FLAGS "-mhvx"
```

Comment:
Are we confident that `-mhvx` is supported by all of the compilers that might build this code?

I'm assuming that _typically_ the clang provided by Hexagon Toolchain will be used.  But I'm a little fuzzy about the intended level of support for other compilers, e.g. a user-supplied build of Clang/LLVM.

Reply:
Would it make sense to update [src/runtime/hexagon/README.md](https://github.com/apache/tvm/blob/15ee9bb5757915c73569f3ebdb5e52a4312663aa/src/runtime/hexagon/README.md) to clarify the version(s) of LLVM that support flags like `-mhvx`?

Or alternatively, use CMake's [CheckCXXCompilerFlag](https://cmake.org/cmake/help/latest/module/CheckCXXCompilerFlag.html) function to see if `-mhvx` is supported, and only use that flag if it is?

Reply:
Thanks for the review @cconvey.

I can add the details in the README or add a CMake check, but the `-mhvx` flag was added to clang all the way back in 2017 in [LLVM 6.0 release](https://releases.llvm.org/6.0.0/docs/ReleaseNotes.html#changes-to-the-hexagon-target) if not earlier, which predates the entire TVM project, so we can also probably assume safely that the `-mhvx` flag will be available for practically anyone building the TVM project now.

If you think it might still be better to add the check or the README change, please let me know which one you think makes more sense and I can make that change. Thanks

Reply:
That makes total sense, I didn't realize `-mhvx` support went back that far.  I agree that there's no need for any additional documentation or checking.